qcengine/programs/nwchem/runner.py

```diff
@@ -179,6 +179,10 @@ def build_input(
 
         if opts.pop("geometry__noautoz", False):
             molcmd = re.sub(r"geometry ([^\n]*)", r"geometry \1 noautoz", molcmd)
+        # someday when minimum >=py38 `if val := opts.pop("geometry__autosym", False):`
```

Comment:
Thanks for the line comment! `:=`a syntax I keep forgetting about (probably because many of my projects are "as many Python versions as possible)

Reply:
This was going to be my first use for `:=` before I remembered qcng isn't >=py38.